### PR TITLE
[azmetrics] update docs and prep for GA

### PR DESCRIPTION
**Pull request by gracewilcox**
* update readme
* update changelog
* prep for release

### CODE REVIEW

File: sdk/monitor/query/azmetrics/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 # Release History
 
-## 0.1.1 (Unreleased)
+## 1.0.0 (2024-04-09)
 
 ### Features Added
+* First GA
```
Comment (chlowell) [inline sdk/monitor/query/azmetrics/CHANGELOG.md:6]: Eh, I'd just delete this section. This isn't really a feature and the version number expresses the same thing.

File: sdk/monitor/query/azmetrics/README.md [hunk 2]
```diff
@@ -73,6 +76,9 @@ comments.
 [azure_sub]: https://azure.microsoft.com/free/
 [cloud_documentation]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud
 [default_cred_ref]: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/azidentity#defaultazurecredential
+[ingestion/azlogs]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/ingestion/azlogs
```
Comment (chlowell) [inline sdk/monitor/query/azmetrics/README.md:79]: I don't know of one that wouldn't, but I'm afraid some markdown parser won't allow `/` in a link label
```suggestion
[ingestion_azlogs]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/ingestion/azlogs
```

File: sdk/monitor/query/azmetrics/README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Azure Monitor Query Metrics client module for Go
 
-The Azure Monitor Query Metrics client module is used to execute read-only queries against [Azure Monitor][azure_monitor_overview]'s metrics platform.
-
-[Metrics][metrics_overview] collects numeric data from monitored resources into a time series database. Metrics are numerical values that are collected at regular intervals and describe some aspect of a system at a particular time. Metrics are lightweight and capable of supporting near real-time scenarios, making them particularly useful for alerting and fast detection of issues.
+* Query metrics (this module)- execute read-only queries against [Azure Monitor Metrics][metrics_overview]
```
Comment (chlowell) [inline sdk/monitor/query/azmetrics/README.md:3]: 🤔 this is probably a job for colon
```suggestion
* Query metrics (this module): execute read-only queries against [Azure Monitor Metrics][metrics_overview]
```

**Review by chlowell — APPROVED** [review]

File: sdk/monitor/query/azmetrics/README.md (hunk 3 above)
Comment (gracewilcox) [reply sdk/monitor/query/azmetrics/README.md]: fixed in [63dda26](https://github.com/Azure/azure-sdk-for-go/pull/22729/commits/63dda26dd58e2aa4a3d108fc27e4cbfa30bfd77c)

File: sdk/monitor/query/azmetrics/README.md (hunk 2 above)
Comment (gracewilcox) [reply sdk/monitor/query/azmetrics/README.md]: fixed in [63dda26](https://github.com/Azure/azure-sdk-for-go/pull/22729/commits/63dda26dd58e2aa4a3d108fc27e4cbfa30bfd77c)

File: sdk/monitor/query/azmetrics/CHANGELOG.md (hunk 1 above)
Comment (gracewilcox) [reply sdk/monitor/query/azmetrics/CHANGELOG.md]: fixed in [63dda26](https://github.com/Azure/azure-sdk-for-go/pull/22729/commits/63dda26dd58e2aa4a3d108fc27e4cbfa30bfd77c)

**Pull request merged**